codegen.yml

```diff
@@ -60,6 +60,7 @@ generates:
       - typescript
       - typescript-operations
       - typescript-react-apollo
+      - fragment-matcher
```

Comment:
generates possibleTypes so that we can use fragments and nesting.

Prob can move editor to this at some point too, then the startup of the editor is a bit faster, and in case the API is down we are not getting a blank page.